pre_commit_mirror_maker/main.py

```diff
@@ -70,16 +71,30 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
     )
     args = parser.parse_args(argv)
 
+    minimum_pre_commit_version = '0'
+
+    if args.types_or:
+        match_key = 'types_or'
+        match_val = f'[{args.types_or}]'
+        minimum_pre_commit_version = '2.9.0'
```

Comment:
2.9.2 is a slightly better version, `types_or` was subtly broken in the initial release